system_modules/napsequence/src/sequenceeditor.h

```diff
@@ -17,6 +17,7 @@
 #include <parameter.h>
```

Comment:
Missing MPL2 license

Reply:
I'm seeing the license on top of the file.. did you add it ?